orchestration/dagster_orchestration/hca_orchestration/config/resources/target_hca_dataset/test.yaml

```diff
@@ -0,0 +1,4 @@
+dataset_name: fake_hca_dataset_name
```

Comment:
i don't like the pattern of preconfiguring config resources for tests like this, since the config you're specifying is specific to a particular pipeline run. these are essentially test fixtures, and they probably shouldn't be living alongside application config.